contributing.md

```diff
@@ -13,7 +13,7 @@ As you use Bazel, you may find things that can be improved.
 You can help by [reporting issues](http://github.com/bazelbuild/bazel/issues)
 when:
 
-   - Bazel requires a crash or correctness bug.
+   - There is a crash or for Bazel correctness.
```

Comment:
I actually don't get this sentence. @aiuto - I know what reporting a crash is, but I am not sure what a "correctness bug" is. Can you explain?

In the meantime, I think we can rephrase this list item to:
 - Bazel crashes

Reply:
Done.